### PR TITLE
Add REACH to Base

### PR DESCRIPTION
**Pull request by Hiich**
The whole /reach protocol is migrating to base using AA

### CODE REVIEW

File: data/$Reach/data.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "decimals": 18,
+  "description": "The /Reach platform fosters a collaborative environment where creators and contributors unite to enhance the visibility of Web3 projects, while also reaping rewards for their participation.",
+  "website": "https://getreach.xyz",
+  "twitter": "https://twitter.com/GetReachxyz",
+  "tokens": {
+    "ethereum": {
+      "address": "0x8b12bd54ca9b2311960057c8f3c88013e79316e3"
+    },
+    "base": {
+      "address": "0x4379c13143eb91148ff9282cfb2f93536687a45b"
```
Comment (ymittal) [inline data/$Reach/data.json:13]: Is this a valid Base contract address?

**Comment by Hiich** [comment]
@cfluke-cb

File: data/$Reach/data.json (hunk 1 above)
Comment (Hiich) [reply data/$Reach/data.json:13]: Yes it is : https://basescan.org/address/0x4379c13143Eb91148fF9282CFb2f93536687A45b#internaltx
it has been deployed using base-migrate.xyz

**Review by cfluke-cb — APPROVED** [review]
Looks good on Base side.

**Review by annieke — APPROVED** [review]

**Pull request merged**